### PR TITLE
Docs: Document 'todo' in log details object

### PR DESCRIPTION
**Pull request by Arkni**
I don't know if this worth the mention, but I found it useful to document it in case someone want to create a custom reporter.

### CODE REVIEW

**Comment by platinumazure** [comment]
Question: Should we document `skip` here as well? (I'm not sure if skipped tests are logged.)

**Comment by Arkni** [comment]
The callbacks that passed to `skipped` tests are ignored by `QUnit` and replaced by `noop`s. So no, they are not logged.

File: docs/callbacks/QUnit.log.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ The properties of the details argument are listed below as options.
 | `module` (string) | The test module name of the assertion. If the assertion is not connected to any module, the property's value will be _undefined_. |
 | `name` (string) | The test block name of the assertion. |
 | `runtime` (number) | The time elapsed in milliseconds since the start of the containing [`QUnit.test()`](/QUnit/test), including setup. |
+| `todo` (boolean) | Indicates whether or not this assertion was part of a todo test |
```
Comment (trentmwillis) [inline docs/callbacks/QUnit.log.md]: Nitpick: Can you add a period to the end of the description? All the other descriptions follow that convention.

**Comment by Arkni** [comment]
Done!

**Comment by trentmwillis** [comment]
Thanks!

**Pull request merged**